micarraylib/core.py

```diff
@@ -66,15 +66,16 @@ class Dataset:
     """
 
     def __init__(
-        self, name, fs, array_format, capsule_coords, download=True, data_home=None
+        self, name, fs, array_format, capsule_coords, download=True, data_home=None, partial_download=None,
+            force_overwrite=False, cleanup=False
```

Comment:
The [sowndata api](https://github.com/soundata/soundata/blob/40a9527c4e6864b9635fcc2b68f14a28ae1b41a2/soundata/core.py#L298) requests `partial_download=None, force_overwrite=False, cleanup=False`, which matches your default implementation. :+1: